src/create-release.js

```diff
@@ -18,6 +18,7 @@ async function run() {
     const body = core.getInput('body', { required: false });
     const draft = core.getInput('draft', { required: false }) === 'true';
     const prerelease = core.getInput('prerelease', { required: false }) === 'true';
+    const commitish = core.getInput('commitish', { required: false }) || context.sha;
```

Comment:
To make it behave same without the input value as it behaved before this change - i.e. just let the git API use it's default:

```suggestion
    const commitish = core.getInput('commitish', { required: false });
```

Reply:
Yes, that *is* an option, but I don't think it's what most users of this action would expect. My suggested change doesn't inconvenience hobbyists who only commit and build on master, since `context.sha` will still resolve to head of master in those cases, but it also doesn't confuse more advanced users who build on multiple branches, since it will release on the head of those actual branches, and not a totally different branch.

As I wrote on #31:

> After looking into the code, I realized that the issue is that this action calls [octokit.repos.createRelease()](https://octokit.github.io/rest.js/#octokit-routes-repos-create-release) without a `target_commitish`, which makes the API default to `the repository's default branch (usually master)`. I created #32 to set `target_commitish` with what I think is a more reasonable default in the context of a CI workflow.

So yeah, I'm a bit reluctant to make that change, but still, being able to actually set a commitish is better than the current situation.

Reply:
@fleskesvor, you are 100% correct.

Correct me if I'm wrong, `context.sha` is the same as repository's `github.sha`? If so, I don't see any problem with that since github.sha uses long commit-sha afaik.